person.rb

```diff
@@ -8,10 +8,11 @@ def initialize(age, name = 'Unknown', parrent_permission: true)
     @name = name
     @age = age
     @parrent_permission = parrent_permission
+    @rentals = []
   end
 
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] Kindly add `:rentals` to attr_accessor as it is required to have `Getters` and `Setters` for @rentals

Reply:
I think I had this one already.